charts/ocis/templates/settings/deployment.yaml

```diff
@@ -42,6 +42,14 @@ spec:
             - name: SETTINGS_DEBUG_PPROF
               value: {{ .Values.debug.profiling | quote }}
 
+            # cache
+            - name: SETTINGS_CACHE_STORE
+              value: {{ default "memory" .Values.cache.type | quote }}
+            {{- if ne (default "memory" .Values.cache.type) "noop" }}
+            - name: SETTINGS_CACHE_STORE_NODES
```

Comment:
Does `memory` also need to know the nodes?

Reply:
fixed, please have a look again

Reply:
Should we replace `has` by `mustHas`? From the [documentation](https://helm.sh/docs/chart_template_guide/function_list/#has-musthas): 

> has panics if there is a problem, while mustHas returns an error to the template engine if there is a problem.